mon-pix/app/styles/pages/_user-certifications-get.scss

```diff
@@ -6,9 +6,6 @@
   }
 }
 
-.user-certifications-page-get_link-to {
-  display: inline-block;
-  padding: 20px 0;
-  font-size: 1.5rem;
-  color: $pix-blue;
+.link__return-to {
```

Comment:
ici tu redéfinis une classe déjà définie. C'est difficile de s'y retrouver. C'est mieux de définir une nouvelle classe, propre au contexte où se trouve ce bouton. Par exemple, `.user-certifications-page-get__link-return-to`. Selon la nomenclature BEM, ce qui est avant `__` représente le block (donc la page dans laquelle s'inscrit l'élément), et après `__`, l'élément en question (donc ici le lien de retour).

Reply:
Je trouvais un peu lourd de redéfinir une classe avec la nomenclature BEM, mais c'est surement plus clair en effet. Je modifie ça :)